python_modules/dagster/dagster/core/storage/runs/schema.py

```diff
@@ -74,6 +74,12 @@
     db.Column("body", db.Text),
 )
 
+TelemetryTable = db.Table(
```

Comment:
only comment here is maybe this name is too specific.  Maybe this should be called instance or something like that?  

I'd also want to know if we'd want to track changes to the dagster.yaml...

Reply:
yea we could hash it